src/index.ts

```diff
@@ -71,6 +75,44 @@ const discordUserAnnouncementDictionary: { [key: string]: string } = {
   'NutDragSwag#0374': 'alexander_the_great.mp3'
 };
 
+// Event triggered when a message is sent in a text channel
+client.on('messageCreate', async (message) => {
+  // Commands are represented by a '!'
+  if (message.content.charAt(0) == '!') {
+    const userCommand = message.content.split('!')[1].toLowerCase();
+    channel = message.member?.voice.channel as VoiceBasedChannel;
+
+    if (channel) {
+      if (userCommand === 'cmere') {
+        const connection = await connectToChannel(channel);
+        connection?.subscribe(audioPlayer);
+      } else if (userCommand === 'gtfo') {
+        // Stop the audio player if it's playing. This will cause the bot
+        // to disconnect from the voice channel as well
+        if (audioPlayer.state.status === AudioPlayerStatus.Playing) {
+          audioPlayer.stop(true);
+
+          // Return when the audio player signals it's idle
+          await entersState(audioPlayer, AudioPlayerStatus.Idle, 5000);
+          return;
+        } else {
+          // If not playing anything, simply disconnect from the voice channel
+          const connection = getVoiceConnection(channel.guild.id);
+          connection?.destroy();
+        }
+      } else {
+        fs.readdir('./clips', function(err, files) {
```

Comment:
This is perfectly fine to write this way but for consistency with the rest of the app it might be nice to refactor 104 and 105 to be arrow functions instead e.g.

`fs.readdir('./clips', (err, files) => {`

Reply:
Much better to be consistent. Fixed.